src/context/basket.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import ItemDetails from './item.js'
+
+const ContextBasket = ({children}) => {
+    return(
+        <>
+            <ItemDetails>
```

Comment:
This isn't correct, here you're just passing the provider for items into your basket state so they will always have the same state